src/chapters/booleans-and-conditionals/conditionals.rst

```diff
@@ -0,0 +1,233 @@
+Conditionals
+============
+
+.. index:: ! conditional
+
+A the beginning of this chapter, we motivated our exploration of booleans in JavaScript by observing that having a formal notion of truth and falsehood would allow us to write code that executes conditionally. In other words, we want to be able to write code that only executes when a given condition is ``true``.
+
+Again, here is our motivating example:
+
+.. admonition:: Example
+
+   Consider a banking application that can remind you when a bill is due. The application will notify the user that a bill is due soon, but *only if* the bill has not already been paid.
+
+We summarized the condition as follows: Send a notification of an upcoming bill if the statement "the bill is unpaid" is true.
+
+In such a program, booleans would allow us to write JavaScript that represents the conditional "the bill is unpaid", and based on the truth of such a statement, execute or skip code that notifies the user of an upcoming due date.
+
+The JavaScript construct that enables such behavior is a **conditional**.
+
+``if`` Statements
+-----------------
+
+.. index:: ! if, code block
+
+The most basic form of a conditional is an **if statement**. Here's how to create one in JavaScript:
+
+.. figure:: figures/if.png
+   :height: 400px
+   :alt: The structure of a conditional with an if statement
+
+Let's look at each component of this new syntax.
+
+- The ``if`` statement consists of a header line and a body. The header line begins with the keyword ``if`` followed by a boolean expression enclosed in parentheses. 
+- ``condition`` is a boolean expression, that is, something that evaluates to either ``true`` or ``false``.
```

Comment:
``condition`` is a boolean expression that evaluates to either ``true`` or ``false``.

Reply:
Your rephrasing could be read as implying that this is a special boolean expression that evaluates to `true` or `false`, while others may not. I think a parenthetical statement will clean this up while remaining precise.

Reply:
Ok.